examples/llsec/ipv6-demos/rpl-border-router/Makefile

```diff
@@ -0,0 +1,48 @@
+CONTIKI_PROJECT=border-router
+all: $(CONTIKI_PROJECT) 
+
+CONTIKI=../../../..
+
+#linker optimizations
+SMALL=1
+
+CFLAGS += -DPROJECT_CONF_H=\"project-conf.h\"
+PROJECT_SOURCEFILES += slip-bridge.c
+
+#Simple built-in webserver is the default.
+#Override with make WITH_WEBSERVER=0 for no webserver.
+#WITH_WEBSERVER=webserver-name will use /apps/webserver-name if it can be
+#found in the /apps, /platform/$(TARGET)/apps/, or current directory (in that order).
+# WITH_WEBSERVER=webserver for /apps/webserver
+# WITH_WEBSERVER=raven-webserver for /platform/avr-raven/apps/raven-webserver/
+#make clean before changing webservers!
+
+#Note /apps/webserver contains a 2500 byte style sheet which is a severe test
+#of the slip connection. Large MSS together with low baud rates without flow
+#control will overrun the transmit buffer when the style sheet is requested.
+
+WITH_WEBSERVER=0
```

Comment:
This is probably because I disabled TCP in the corresponding `project-conf.h`. Also, note that I did not copy some files that are required for enabling the web server over to this example folder. I could do so if you like.